tokio-net/tests/process_issue_42.rs

```diff
@@ -16,29 +15,28 @@ use tokio_net::process::Command;
 mod support;
 use support::*;
 
-fn run_test() {
+async fn run_test() {
```

Comment:
I'm not sure if I understand why this is an async fn now?

Reply:
Yep doesn't need to be async 👍